vendor.conf

```diff
@@ -48,7 +48,7 @@ github.com/xeipuuv/gojsonschema master
 github.com/xeipuuv/gojsonreference master
 github.com/xeipuuv/gojsonpointer master
 go4.org master https://github.com/camlistore/go4
-github.com/ostreedev/ostree-go aeb02c6b6aa2889db3ef62f7855650755befd460
+github.com/ostreedev/ostree-go 56f3a639dbc0f2f5051c6d52dade28a882ba78ce
```

Comment:
Shouldn’t this be a separate PR? Or is it a necessary component?

Reply:
I can split it to a separate PR, but it is just to silence a compiler warning